Pinta.Core/Effects/Utility.cs

```diff
@@ -194,282 +204,286 @@ public static int FastDivideShortByByte (ushort n, byte d)
+		0xA57EB503, 0x00000000, 39, // 198
+		0x5254E78F, 0x00000000, 38, // 199
+		0x51EB851F, 0x00000000, 38, // 200
+		0x028C1979, 0x00000000, 33, // 201
+		0xA237C32B, 0xA237C32B, 39, // 202
+		0xA16B312F, 0x00000000, 39, // 203
+		0xA0A0A0A1, 0x00000000, 39, // 204
+		0x4FEC04FF, 0x00000000, 38, // 205
+		0x9F1165E7, 0x9F1165E7, 39, // 206
+		0x27932B49, 0x00000000, 37, // 207
+		0x4EC4EC4F, 0x00000000, 38, // 208
+		0x9CC8E161, 0x00000000, 39, // 209
+		0x27027027, 0x27027027, 37, // 210
+		0x9B4C6F9F, 0x00000000, 39, // 211
+		0x9A90E7D9, 0x9A90E7D9, 39, // 212
+		0x99D722DB, 0x00000000, 39, // 213
+		0x991F1A51, 0x991F1A51, 39, // 214
+		0x4C346405, 0x00000000, 38, // 215
+		0x97B425ED, 0x97B425ED, 39, // 216
+		0x4B809701, 0x4B809701, 38, // 217
+		0x2593F69B, 0x2593F69B, 37, // 218
+		0x12B404AD, 0x12B404AD, 36, // 219
+		0x94F2094F, 0x94F2094F, 39, // 220
+		0x25116025, 0x25116025, 37, // 221
+		0x24E6A171, 0x24E6A171, 37, // 222
+		0x24BC44E1, 0x24BC44E1, 37, // 223
+		0x49249249, 0x49249249, 38, // 224
+		0x91A2B3C5, 0x00000000, 39, // 225
+		0x90FDBC09, 0x90FDBC09, 39, // 226
+		0x905A3863, 0x905A3863, 39, // 227
+		0x47DC11F7, 0x47DC11F7, 38, // 228
+		0x478BBCED, 0x00000000, 38, // 229
+		0x8E78356D, 0x8E78356D, 39, // 230
+		0x46ED2901, 0x46ED2901, 38, // 231
+		0x8D3DCB09, 0x00000000, 39, // 232
+		0x2328A701, 0x2328A701, 37, // 233
+		0x23023023, 0x23023023, 37, // 234
+		0x45B81A25, 0x45B81A25, 38, // 235
+		0x22B63CBF, 0x00000000, 37, // 236
+		0x08A42F87, 0x08A42F87, 35, // 237
+		0x44D72045, 0x00000000, 38, // 238
+		0x891AC73B, 0x00000000, 39, // 239
+		0x88888889, 0x00000000, 39, // 240
+		0x10FEF011, 0x00000000, 36, // 241
+		0x8767AB5F, 0x8767AB5F, 39, // 242
+		0x86D90545, 0x00000000, 39, // 243
+		0x4325C53F, 0x00000000, 38, // 244
+		0x85BF3761, 0x85BF3761, 39, // 245
+		0x85340853, 0x85340853, 39, // 246
+		0x10953F39, 0x10953F39, 36, // 247
+		0x42108421, 0x42108421, 38, // 248
+		0x41CC9829, 0x41CC9829, 38, // 249
+		0x10624DD3, 0x00000000, 36, // 250
+		0x828CBFBF, 0x00000000, 39, // 251
+		0x41041041, 0x41041041, 38, // 252
+		0x81848DA9, 0x00000000, 39, // 253
+		0x10204081, 0x10204081, 36, // 254
+		0x80808081, 0x00000000, 39, // 255
+	};
+
+	/// <summary>Gets the nearest step angle in radians.</summary>
+	/// 
 	/// <returns>The nearest step angle in radians.</returns>
+	/// 
 	/// <param name="angle">Angle in radians.</param>
 	/// <param name="steps">Number of steps to divide the circle.</param>
-	public static double GetNearestStepAngle (double angle, int steps)
+	/// 
+	public static RadiansAngle GetNearestStepAngle (RadiansAngle angle, int steps)
 	{
-		double fullTurn = 2 * Math.PI;
+		if (steps < 1)
+			throw new ArgumentOutOfRangeException (nameof (steps));
+
+		double fullTurn = RadiansAngle.MAX_RADIANS;
 		double stepAngle = fullTurn / steps;
-		int sector = Convert.ToInt32 (Math.Truncate (angle % fullTurn / stepAngle));
+		int sector = Convert.ToInt32 (Math.Truncate (angle.Radians / stepAngle));
```

Comment:
I think this could also use `Math.Round` like the other recent PR?

Reply:
@cameronwhite I just changed that. Is this what you had in mind?